src/us/kbase/workspace/test/database/mongo/ShockBlobStoreIntegrationTest.java

```diff
@@ -44,7 +44,7 @@
 import us.kbase.workspace.database.mongo.ShockBlobStore;
 import us.kbase.workspace.database.mongo.exceptions.NoSuchBlobException;
 
-public class ShockBlobStoreTest {
+public class ShockBlobStoreIntegrationTest {
```

Comment:
I think this classname is far too long, and would benefit from abbreviation to something like "ShockBlbStrIntTest"